docs/developers/talawa-api/mutations.md

```diff
@@ -4,8 +4,10 @@ title: Mutations
 ---
 
 ## Mutations According to Roles
-This doucument outlines our mutations and to which type of user each is constrained
-### Type Of Users in Talawa 
+
+This document outlines our mutations and to which type of user each is constrained
+
+### Type Of Users in Talawa
 
 There are mainly 3 types of User types in talawa -
```

Comment:
Uncapitalize User and capitalize talawa please

Reply:
DOne